roles/core/dns_server/vars/CentOS_7.yml

```diff
@@ -1,6 +1,7 @@
 ---
```

Comment:
If the 4 files {CentOS,RedHat}_{7,8}.yml files should be identical, I would revert to RedHat.yml.

Reply:
Impossible because ipaddr filter. We can stillr educe Centos_7 to RedHat_7 and same for 8 version.